codes/c/include/print_util.h

```diff
@@ -25,7 +25,7 @@ extern "C" {
  * @param arr
  * @param size
  */
-static void printArray(int *arr, int size) {
+static void printArray(int arr[], int size) {
```

Comment:
This modification influence other files such as `array.c`. Have you tested them?

Reply:
I have tested, it works as well.